pkg/types/kustomization.go

```diff
@@ -44,6 +44,10 @@ type Kustomization struct {
 	// URLs and globs.
 	Resources []string `json:"resources,omitempty" yaml:"resources,omitempty"`
 
+	// CRDs specifies the CRD files, either json or yaml format
```

Comment:
please shorten to

> CRDs specifies relative paths to custom resource definition files.

I don't think we mention JSON or YAML anywhere else (e.g. resources, patches), so might as well leave it off here too.